### PR TITLE
Wording for MIT license and external contributors

### PR DESCRIPTION
**Pull request by LefterisJP**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,10 +43,25 @@ See [TODO](https://github.com/ethereum/cpp-ethereum/wiki/TODO)
...
 ### License
 
-See [LICENSE](LICENSE)
+All new contributions are under the [MIT license](http://opensource.org/licenses/MIT).
+See [LICENSE](LICENSE). Some old contributions are under the [GPLv3 license](http://www.gnu.org/licenses/gpl-3.0.en.html). See [GPLV3_LICENSE](GPLV3_LICENSE).
 
 ### Contributing
 
+All new contributions are added under the MIT License. Please refer to the `LICENCSE` file in the root directory.
+To state that you accept this fact for all of your PRs please add yourself to the list of external contributors
```
Comment (chriseth) [inline README.md]: Perhaps abstract from the technical side of how contributions are introduced into the codebase (i.e. replace `PRs` by `contributions`), or at least spell out `PR`.

File: README.md [hunk 2]
```diff
@@ -43,10 +43,25 @@ See [TODO](https://github.com/ethereum/cpp-ethereum/wiki/TODO)
...
+like in the example below.
+
+#### External Contributors
+I hereby place all my contributions in this file under an MIT
```
Comment (chriseth) [inline README.md]: `file` -> `codebase`?

File: README.md (hunk 1 above)
Comment (LefterisJP) [inline README.md]: good point

File: README.md [hunk 3]
```diff
@@ -43,10 +43,25 @@ See [TODO](https://github.com/ethereum/cpp-ethereum/wiki/TODO)
 
 ### License
 
-See [LICENSE](LICENSE)
+All new contributions are under the [MIT license](http://opensource.org/licenses/MIT).
+See [LICENSE](LICENSE). Some old contributions are under the [GPLv3 license](http://www.gnu.org/licenses/gpl-3.0.en.html). See [GPLV3_LICENSE](GPLV3_LICENSE).
 
 ### Contributing
 
+All new contributions are added under the MIT License. Please refer to the `LICENCSE` file in the root directory.
```
Comment (subtly) [inline README.md]: LICENSE

**Comment by gavofyork** [comment]
windows failing?

**Comment by LefterisJP** [comment]
WoW, something actually depended on this file. Helped us find a Typo :+1:

**Pull request merged**